src/sentry/utils/services.py

```diff
@@ -465,6 +470,64 @@ def setup(self) -> None:
             backend.setup()
 
 
+KeyFetch = Callable[[Context, str, Mapping[str, Any]], str | int]
+
+
+def make_writebehind_selector(
+    *, option_name: str, key_fetch: KeyFetch, move_to: str, move_from: str
```

Comment:
Just for people who might not be as familiar, might be worth defining that move_to/from refer to the backend keys